.github/workflows/e2e.yml

```
@@ -56,20 +52,22 @@ jobs:
             **/node_modules
             **/.turbo
             /home/runner/.cache/Cypress
-          key: ${{ runner.os }}-node-${{ matrix.node-version }}-${{ hashFiles('**/package-lock.json') }}
+            ~/.pnpm-store
+          key: ${{ runner.os }}-node-${{ matrix.node-version }}-${{ hashFiles('**/pnpm-lock.yaml') }}
```

actionlint (1.7.7)

property "node-version" is not defined in object type {}

(expression)